README.md

```diff
@@ -1,38 +1,85 @@
+
+### Overview
+Our goal is to help casual users understand and visualize where their computer packets are going and where they are coming from. We hope to have our website be both informative and interactive. 
+
+On the technical front, this means that to build out the PCAP analysis feature of our website, we will need to be able to do the following with our users’ files:
+
+1. File Upload / Data Collection ― since our main source of data are PCAP files, we can either enable PCAP file uploads or retrieve a PCAP file from a user-inputted URL.
+
+2. File Parsing / Data Processing ― we want to be able to extract key attributes from uploaded packets, including packet source, destination, protocol, and time. 
+
+3. File Storage / Data Retrieval ― in order to retrieve the above attributes, we need a way to store our packet information with easy access.
+
+4. File Analysis / Visualizations ― to present our users with their packet information and activity, we will put together a series of intuitive visualizations. These include:
+   * Geographical Map ― showing the country or region of packet source/destination
+   * Frequent Connections ― enumerating the most common connections made
+   * Website Security ― comparing protocols of websites visited by user
+
+
+### Team
 Authors: handeland@google.com, jevingu@google.com, mnuzen@google.com 
 Reviewers: arunkaly@, promanov@
 
+### Built With
+- [Maven](https://maven.apache.org)
+- [Java](https://java.com/en/download/faq/java8.xml)
+- [App Engine](https://cloud.google.com/appengine/docs/standard/java)
+
 ## Setup: 
 1. git clone git@github.com:mnuzen/step-capstone-2020.git
-2. `gsutil cp -r gs://erik-jevin-melba-step-2020 resources/` if you have access to the Google bucket,
-or download the Maxmind Geolite2 Country DB and put it into the resources folder. Any files you would like
-to have available by default should be uploaded to the `files/` subfolder of the resources folder.
+2. download the Maxmind Geolite2 Country DB and put it into the resources folder. Any files you would like to have available by default should be uploaded to the `files/` subfolder of the resources folder.
 3. Setup your [GAE credentials](https://cloud.google.com/docs/authentication/production)
 4. `mvn package appengine:run -Dmaven.test.skip=true`
 
-## Contributing: 
-Before opening a PR, make sure to consult with us through email, or on Github. We have starter issues tagged
-with `good first issue`.
-
-## To add sample PCAP files: 
+### Adding PCAP files: 
 
-1. Download `file1.pcap` and `file2.pcap` from GDrive's PCAP Files folder.
-2. Put in repo under `resources/files/file1.pcap` and `resources/files/file2.pcap`, respectively. 
+1. Place any files inside of the  `resources/files` folder
+2. Update `pcap-uploader.html` to include the path to any added PCAP files
 
-## To add IP information for PCAP parser testing:
+### To add IP information for PCAP parser testing:
 
 1. Download `file2.txt` from GDrive's PCAP Files folder.
```

Comment:
Should we upload the test files? otherwise the project will fail to load

Reply:
I think they've been uploaded right? Which ones are missing?

Reply:
They were uploaded to the cloud bucket, but not GitHub.